src/transformers/models/fuyu/modeling_fuyu.py

```diff
@@ -261,7 +261,9 @@ def forward(
         if inputs_embeds is None:
             inputs_embeds = self.language_model.get_input_embeddings()(input_ids)
             if image_patches is not None and past_key_values is None:
-                patch_embeddings = self.vision_embed_tokens(image_patches.to(self.vision_embed_tokens.weight.dtype))
+                patch_embeddings = self.vision_embed_tokens(
```

Comment:
Let’s try to split in two lines. Also if we can register a buffer for auto device placement or whatever (tried this when porting) would be more readable

Reply:
Let's keep the split in two lines. I prefer not to introduce a variable for auto device placement.